packages/strapi-design-system/src/Accordion/AccordionToggle.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DropdownIcon from '@strapi/icons/FilterDropdown';
+import styled from 'styled-components';
+import { H3, Text } from '../Text';
+import { useAccordion } from './AccordionContext';
+import { Box } from '../Box';
+import { Row } from '../Row';
+import { DropdownIconWrapper } from './DropdownIconWrapper';
+
+const ToggleButton = styled.button`
+  border: none;
+  background: transparent;
+  display: block;
+  width: 100%;
+  text-align: unset;
+  padding: 0;
+`;
+
+export const AccordionToggle = ({ title, description, as, variant, togglePosition, action, ...props }) => {
```

Comment:
What is the utility of the `as` prop? Will it be used to update the heading level only?

Reply:
If so, WDYT about renaming that prop to something like `headingLevel={2}`? And create a mapping `1 => h1`, `2 -> h2` to reduce the scope of that prop... 🤔

Reply:
We can totally 😊 But what if it's not a heading?